COCOA/cmake/Modules/FindHepMC.cmake

```diff
@@ -4,26 +4,31 @@
 #
```

Comment:
Nope. HePMC3 has its own config file. no need for any of those changes. the file can be deleted if hepmc3 support is not planned.